### ISSUE TITLE
Regarding the Download Issue of LAION-5B Pre-trained Weights

### ISSUE THREAD

**Issue by DaIGaN2019** [GuangyuWangLab2021/Loki#38]
Dear Author,
Thank you for sharing your wonderful work and for maintaining this elegant project with such dedication. In your previous response, you provided the download link for LAION-5B. However, when I accessed the link and searched through it, I did not find the pre-trained weights of LAION-5B under the COCA framework. Using "coca_" as the search term, I found the following 4 pre-trained weights. Could you confirm if one of these 4 is the one I need?
Additionally, in your paper, you mentioned that the training was conducted with settings of batch size = 64 and epochs = 20. Could you please let me know if all other training parameters were kept as default?Any response would be greatly appreciated!

With my sincere best wishes！

<img width="637" height="583" alt="Image" src="https://github.com/user-attachments/assets/78acfc45-56af-4374-adad-0197ab040d72" />

<img width="625" height="108" alt="Image" src="https://github.com/user-attachments/assets/22cacffb-eacc-4c88-8945-5584d2fade03" />

<img width="761" height="90" alt="Image" src="https://github.com/user-attachments/assets/2daba562-8307-41f1-a259-5189814850e9" />

**Comment by osakemon** [comment]
Dear DaIGaN2019,

Apologies for the delayed response.

We used the coca_ViT-L-14 pretrained weights with the following hyperparameters:
warmup: 1000
learning rate: 5e-6
weight decay: 0.1

Please feel free to reach out if you have any additional questions.

Best regards,
Weiqing
